modules/015-admission-policy-engine/crds/doc-ru-security-policy.yaml

```
@@ -66,133 +66,133 @@ spec:
                     allowedAppArmor:
                       type: array
                       description: |
-                        Список допустимых к использованию контейнерами профилей AppArmor.
-                      example: ["runtime/default", "unconfined"]
+                        Список профилей AppArmor для контейнера.
+                      example: [runtime/default, unconfined]
```

```suggestion
                      example: ["runtime/default", "unconfined"]
```
Был валидный  JSON, стал не валидный. Спеку надо править очень акуратно. Давайте не будем выходить за рамки description, т.к. это может нарушить работу DKP.

@MariaMakeeva @elenashliaga FYI